browser/js/cart/cart.js

```diff
@@ -45,4 +45,12 @@ app.factory('cartFactory', function ($http) {
 			});
 		}
 	};
-});
\ No newline at end of file
+});
+
+// Filter for cents -> dollars
+
+app.filter("centsToDollars", function() {
```

Comment:
Yes absolutely, in the next story :). A question to decide though - are we going to let people purchase multiple of each movie? We'll have to update our models a bit to accommodate that then...